content/integrations/redis.md

```diff
@@ -1,36 +1,84 @@
 ---
 title: Datadog-Redis Integration
 integration_title: Redis
-
+newhlevel: true
 kind: integration
 git_integration_title: redis
+updated_for_agent: 5.8.5
 ---
 
-### Overview
-
-Track and graph your Redis activity and performance metrics with slice-and-dice at all levels from individual column families to entire clusters.
+# Overview
 
 ![Redis default dashboard](/static/images/redis.png)
 
-Learn more about how to monitor Redis performance metrics thanks to [our series of posts](https://www.datadoghq.com/blog/how-to-monitor-redis-performance-metrics/). We detail the key performance metrics, how to collect them, and how to use Datadog to monitor Redis.
+Track and graph your Redis activity and performance metrics with slice-and-dice at all levels
+from individual column families to entire clusters.
+
+Learn more about how to monitor Redis performance metrics thanks to
+[our series of posts](https://www.datadoghq.com/blog/how-to-monitor-redis-performance-metrics/).
+We detail the key performance metrics, how to collect them, and how to use Datadog to monitor Redis.
+
+# Configuration
+
+1.  Configure the Agent to connect to the Redis server. Edit conf.d/redisdb.yaml:
+
+        init_config:
+
+        instances:
+          - host: localhost
+            port: 6379
+            tags:
+              - optional_tag1
+              - optional_tag2
+            keys:
+              - key1
+    {:.language-yaml}
+
+1.  Restart the Agent
+
+## Configuration Options
+
+* `unix_socket_path` - (Optional) - Can be used instead of `host` and `port`.
+* `db`, `password`, and `socket_timeout` - (Optional) - Additional connection options.
+* `warn_on_missing_keys` - (Optional) - Display a warning in the info page if the keys we're tracking are missing.
+* `slowlog-max-len` - (Optional) - Maximum number of entries to fetch from the slow query log. By default, the check will
+        read this value from the redis config. If it's above 128, it will default to 128 due to potential increased latency
+        to retrieve more than 128 slowlog entries every 15 seconds. If you need to get more entries from the slow query logs
+        set the value here. Warning: It may impact the performance of your redis instance
+* `command_stats` - (Optional) - Collect INFO COMMANDSTATS output as metrics.
```

Comment:
Generally like this level of detail.  Should we confider this in other integrations as well?